### PR TITLE
[DEVX-1882] Add new Management APIs tile.

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description
[Jira ticket](https://nexmoinc.atlassian.net/browse/DEVX-1882)

Create a "proper" tile with the Global and Developer APIs

### CODE REVIEW

File: app/views/static/_products.html.erb [hunk 1]
```diff
@@ -280,26 +280,27 @@
...
+      <div class="Vlt-card__image Vlt-card__image--app Vlt-card__image--green-light">
+        <div class="Vlt-card__image__icon">
+          <svg>
+            <use xlink:href="/symbol/volta-icons.svg#Vlt-icon-laptop"/></svg>
+        </div>
+      </div>
+      <div class="Vlt-card__content">
+        <h2>Management APIs and Services</h2>
+        <nav>
+          <a href="/api/developer/messages">Messages</a>
```
Comment (tbedford) [inline app/views/static/_products.html.erb]: This is actually probably better termed "Message Search" rather than "Messages".

**Comment by mheap** [comment]
Screen Shot 2019-11-29 at 13 30 21

Is there a way to keep it the same height as all of the other tiles?

**Comment by tbedford** [comment]
> Is there a way to keep it the same height as all of the other tiles?

Possibly related - could it just be "Management APIs" and lose the "and Services."

**Comment by tbedford** [comment]
Also, I just noticed, above "Account" there is some small text that says "Management". The box on the right does not have that, and also does that mean Account should be under the Management APIs box?

**Comment by mheap** [comment]
Account can be dropped from the Management APIs tile

**Comment by fabianrbz** [comment]
I pushed some changes, the tiles now have the same height.
@tbedford the title was a specific requirement, but we can talk about it in the [Jira ticket](https://nexmoinc.atlassian.net/browse/DEVX-1882)

**Comment by mheap** [comment]
The other tiles look like they're an odd height now with whitespace under `API Reference`. We can change this tile to just `Management APIs` if it helps

**Comment by fabianrbz** [comment]
did some changes, now the tiles have all the same height and it's almost the one they had before

**Comment by fabianrbz** [comment]
@tbedford can you take another look at this?

**Comment by tbedford** [comment]
Still think it's slightly odd that Account is in a management tile and there's a  management API in the next tile over. But....that's another issue...

**Pull request merged**